packages/flutter_tools/test/general.shard/dart_plugin_test.dart

```diff
@@ -492,139 +517,15 @@ void main() {
           'url_launcher',
         };
 
-        final List<PluginInterfaceResolution> resolutions = resolvePlatformImplementation(<Plugin>[
```

Comment:
Why is this test being removed?

Reply:
It just looks like it was removed, because of the git diff is not showing it correctly. It is present here: https://github.com/flutter/flutter/pull/137040/files#diff-2fb2f7fe1d44f737a3b8e0fefdf353e06196290b8461daf7e2b7c33cb615f6a5R582
In the code editor it is much more clear that only the formatting and the `selectDartPluginsOnly: true` flag has changed for all the tests.